jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/deadlines/notifications/impl/NotificationListenerManager.java

```diff
@@ -30,18 +32,59 @@
  *
  */
 public class NotificationListenerManager {
-    
+
+    public static final String KIE_LISTENER_EXCLUDE = "org.kie.jpbm.notification_listeners.exclude";
+
+    public static final String KIE_LISTENER_INCLUDE = "org.kie.jpbm.notification_listeners.include";
+
     private static final Logger logger = LoggerFactory.getLogger(NotificationListenerManager.class);
-    
+
     private static ServiceLoader<NotificationListener> listenersLoaded = ServiceLoader.load(NotificationListener.class);
-    
+
     private static NotificationListenerManager INSTANCE = new NotificationListenerManager();
-    
+
     private List<NotificationListener> listeners = new ArrayList<NotificationListener>();
-    
+
+    private List<String> excludeLists;
+
+    private List<String> includeLists;
+
+    public static List<String> propertyToList(String propertyName) {
+        String name = System.getProperty(propertyName);
+        if (name == null) {
+            return null;
```

Comment:
Why not return `Collections.emptyList()` ?
Then, at createPredicate you can check against "is not empty"...

Reply:
not equivalent... empty string is not equivalent to null. Think aboud include = ""... this will exclude everthing as the predicate will never test true.